src/sidebar/markdown-commands.js

```diff
@@ -134,7 +135,7 @@ export function convertSelectionToLink(state, linkType) {
  * @param {EditorState} state - The current state of the input field.
  * @param {string} prefix - The prefix to add or remove
  *                          before the selection.
- * @param {string?} suffix - The suffix to add or remove after the selection,
+ * @param {string|undefined} suffix - The suffix to add or remove after the selection,
```

Comment:
`string?` is shorthand for `string|undefined|null` but the body of the code only handled the `string` and `undefined` cases properly.

Reply:
Helpful info, thanks.